tests/pfc/test_pfc_pause_lossy.py

```diff
@@ -0,0 +1,106 @@
+import time
+import pytest
+
+from abstract_open_traffic_generator.result import FlowRequest
+from abstract_open_traffic_generator.control import *
+
+from tests.common.helpers.assertions import pytest_assert
+
+from tests.common.reboot import logger
+from tests.common.fixtures.conn_graph_facts import conn_graph_facts,\
+    fanout_graph_facts 
+
+from tests.common.ixia.ixia_fixtures import ixia_api_serv_ip, \
+    ixia_api_serv_user, ixia_api_serv_passwd, ixia_dev, ixia_api_serv_port,\
+    ixia_api_serv_session_id, api
+
+from files.configs.pfc import lossy_configs, one_hundred_gbe, serializer
+from files.configs.pfc import start_delay, traffic_duration, pause_line_rate,\
+    traffic_line_rate, port_bandwidth, bw_multiplier, frame_size
+from files.qos_fixtures import lossless_prio_dscp_map
+
+START_DELAY = [1]
+TRAFFIC_DURATION = [3]
+PAUSE_LINE_RATE = [100]
+TRAFFIC_LINE_RATE = [50]
+BW_MULTIPLIER = [1000000]
+FRAME_SIZE = [1024]
+TOLERANCE_THRESHOLD = .97
+
+@pytest.mark.parametrize('start_delay', START_DELAY)
+@pytest.mark.parametrize('traffic_duration', TRAFFIC_DURATION)
+@pytest.mark.parametrize('pause_line_rate', PAUSE_LINE_RATE)
+@pytest.mark.parametrize('traffic_line_rate', TRAFFIC_LINE_RATE)
+@pytest.mark.parametrize('bw_multiplier', BW_MULTIPLIER)
+@pytest.mark.parametrize('frame_size', FRAME_SIZE)
+
+def test_pfc_pause_lossy_traffic(api, 
+                                 duthost, 
+                                 lossy_configs, 
+                                 start_delay,
+                                 pause_line_rate,
+                                 traffic_line_rate, 
+                                 traffic_duration,
+                                 port_bandwidth,
+                                 frame_size) :
+    """
+    This test case checks the behaviour of the SONiC DUT when it receives 
+    a PFC pause frame on lossy priorities.
+
+                                +-----------+
+    [Keysight Chassis Tx Port]  |           | [Keysight Chassis Rx Port]
+    --------------------------->| SONiC DUT |<---------------------------
+    Test Data Traffic +         |           |  PFC pause frame on 
+    Background Dada Traffic     +-----------+  "lossy" priorities.
+
+    1. Configure SONiC DUT with multipul lossless priorities. 
+    2. On SONiC DUT enable PFC on several lossless priorities e.g priority 
+       3 and 4.
+    3. On the Keysight chassis Tx port create two flows - a) 'Test Data Traffic'
+       and b) 'Background Data traffic'.
+    4. Configure 'Test Data Traffic' such that it contains traffic items
+       with all lossy priorities.
+    5. Configure 'Background Data Traffic' it contains traffic items with
+       all lossless priorities.
+    6. From Rx port send pause frames on all lossless priorities. Then
+       start 'Test Data Traffic' and 'Background Data Traffic'.
+    7. Verify the following: 
```

Comment:
The verification is wrong. The right description is 'When all the traffic items stop, the Rx port receives all the packets of test traffic and background traffic. And their throughput are not impacted by the PFC pause storm'.